scripts/tag-submodules.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
```

Comment:
is this needed? Don't we do this tag from freedom-e-sdk?

Reply:
Accidentally copied in, it's been removed.